batchflow/notifier.py

```diff
@@ -91,13 +91,21 @@ class Notifier:
         Total size of drawn figure.
     savepath : str
         Path to save image, created by tracking entities with `graphs`.
+    disable : bool
+        Whether to disable the notifier completely: progress bar, monitors and graphs.
     *args, **kwargs
         Positional and keyword arguments that are used to create underlying progress bar.
     """
     def __init__(self, bar=None, *args, update_total=True,
                  total=None, batch_size=None, n_iters=None, n_epochs=None, drop_last=False, length=None,
                  frequency=1, monitors=None, graphs=None, file=None,
-                 window=None, layout='h', figsize=None, savepath=None, **kwargs):
+                 window=None, layout='h', figsize=None, savepath=None, disable=False, **kwargs):
+        # Turn off everything if `disable`
+        if disable:
```

Comment:
Why not to add `disable` method?

Reply:
It is supposed to be used for external parametrization like:
```python
def my_func(*args, bar=True, **kwargs)
    ...
    notifier = Notifier(..., disable=not bar)
```